src/renderer/qssWidget/js/qssOfficeWidget.js

```diff
@@ -0,0 +1,391 @@
+                    },
+                    handlerType: "gpii.qssWidget.office.presenter",
+                    markup: null,
+                    styles: {
+                        disabled: "disabled"
+                    },
+                    events: {
+                        onItemFocus: null
+                    },
+                    listeners: {
+                        "onCreate.enable": {
+                            this: "{that}.container",
+                            method: "removeClass",
+                            args: ["{that}.options.styles.disabled"]
+                        }
+                    }
+                }
+            },
+            closeTimer: {
+                type: "gpii.app.timer",
+                options: {
+                    listeners: {
+                        "onTimerFinished.closeWidget": {
+                            func: "{qssWidget}.close",
+                            args: ["{office}.keyboardEvent"]
+                        }
+                    }
+                }
+            }
+        },
+        listeners: {
+            "onCreate.visibilityChange": {
+                funcName: "gpii.qssWidget.office.addVisibilityChangeListener",
+                args: ["{closeTimer}"]
+            },
+            "onDestroy.visibilitychange": {
+                funcName: "gpii.qssWidget.office.removeVisibilityChangeListener"
+            }
+        },
+        invokers: {
+            calculateHeight: {
+                funcName: "gpii.qssWidget.calculateHeight",
+                args: [
+                    "{qssWidget}.container",
+                    "{that}.dom.menuControlsWrapper",
+                    "{that}.dom.heightListenerContainer"
+                ]
+            },
+            close: {
+                funcName: "gpii.qssWidget.office.close",
+                args: [
+                    "{that}",
+                    "{closeTimer}",
+                    "{arguments}.0" // keyboardEvent
+                ]
+            }
+        },
+        events: {
+            onHeightChanged: null
+        }
+    });
+
+
+    /**
+     * Checks the current states and generates the appropriate command
+     * @param {Object} states - simple true/false object with the current states
+     * @param {Object} availableCommands - a simple list of available commands defined in the model
+     * @return {String} - string value of the compiled command
+     */
+    gpii.qssWidget.office.getCommand = function (states, availableCommands) {
+        var stateNames = [],
+            defaultCommand = availableCommands.defaultCommand,
+            resetCommand = availableCommands.resetCommand,
+            allTrue = availableCommands.allTrueCommand,
+            allFalse = availableCommands.allFalseCommand,
+            allStates = 0;
+
+        fluid.each(states, function (state, name) {
+            if (name !== resetCommand) {
+                if (state === true) {
```

Comment:
Can just be if (state) {

Reply:
Done, removed every mention of the forgotten resetCommand. Sorry, my bad 🛩